packages/cactus-plugin-ledger-connector-fabric/src/main/typescript/prometheus-exporter/prometheus-exporter.ts

```diff
@@ -0,0 +1,51 @@
+import promClient from "prom-client";
+import { Transaction, Transactions } from "./response.type";
+import { collectMetrics } from "./data-fetcher";
+import { PrometheusExporterMetricsResponse } from "../generated/openapi/typescript-axios";
+
+export interface PrometheusExporterOptions {
```

Comment:
@jagpreetsinghsasan Any chance to rename this to have an `I` prefix? (convention)

Reply:
Fixed.